styles.scss

```
@@ -96,6 +96,29 @@ $color-link-bold-dark: lighten($color-link-bold, 20%);
       );
     }
   }
+
+  &.adaptive {
+    // For adaptive color schemes, the default is light
```

This seems to be a copy/paste from the above `.dark` class. Is there a way in sass to reuse this, instead of copy/paste?

---

Not a huge deal, we should go ahead and ship this, and just put a TODO. 
```suggestion
    // For adaptive color schemes, the default is light
    // TODO: See about reusing these values in some way
```